(dp1
S'output'
p2
S"<type 'exceptions.NameError'> global name 'b' is not defined"
p3
sS'layer'
p4
S'/home/topf/Web2Py/web2py/applications/Asylum/controllers/asylum.py'
p5
sS'code'
p6
S'<code object <module> at 0xaebfa8d8, file "/home/topf/Web2Py/web2py/applications/Asylum/controllers/asylum.py", line 5>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/topf/Web2Py/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/topf/Web2Py/web2py/applications/Asylum/controllers/asylum.py", line 267, in <module>\n  File "/home/topf/Web2Py/web2py/gluon/globals.py", line 409, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/topf/Web2Py/web2py/applications/Asylum/controllers/asylum.py", line 8, in index\n    b\nNameError: global name \'b\' is not defined\n'
p11
s.